### DISCUSSION TITLE
Cypher: add comments in the language

### DISCUSSION THREAD

**Discussion by vbmithr** [kuzudb/kuzu#1392] [Q&A] [answered]
Very useful when adding comments in text files when writing DDL.

**Comment by acquamarin** [comment, marked as answer]
Hi,

Actually, we currently(version 0.0.2) support writing comments in cypher queries:
1. Single line comment: `//`
2. Multi line comment: `/* */`

In the next release, we are planning to change the single line comment to `--` which follows the SQL standard.
